tool/validate_firebase.py

```diff
@@ -1,30 +1,18 @@
 # Validation tool for firebase collections
 # Relies on an undocumented .collections() API call and a hard coded list of top level collections
 
-import firebase_admin
-from firebase_admin import credentials
-from firebase_admin import firestore
-from core_data_modules.logging import Logger
 import time
-
 import json
 import sys
 import os.path
 
 import validate_nook_model as model
 import validate_nook_model_custom as custom
+import firebase_util
+from firebase_util import firebase_client
```

Comment:
accidentally left over from the previous version?

Reply:
refactored